js/index.js

```diff
@@ -1 +1,90 @@
+
+// MOUSEENTER THE NAVIGATION TO RETURN ITS OPACITY TO 1 TO NEGATE THE ABOVE KEYPRESS OPACITY CHANGE
+navContainer.addEventListener("mouseenter", () => {
+  header.style.opacity = "1"
+  TweenMax.to(".nav-container", 2, {rotation:360, scale:1.5});
+});
+
+// ADDS AN ALERT WHEN THE USER FIRST SCROLLS WITH THE MOUSE WHEEL BUT THEM REMOVES THAT EVENT SO THEY CAN ACTUALLY NAVIGATE THE SITE
+body.addEventListener('wheel', function alertMeOnce(){
+  alert("WEEEEEEEEEEEE!!!!!!");
+  body.removeEventListener('wheel', alertMeOnce);
+});
+
+const funBusBanner = document.querySelector(".intro > img" );
+
+// DRAG THE FUN-BUS INTRO IMAGE TO SCALE IT 150% IN SIZE
+funBusBanner.addEventListener('drag', (e) => e.target.style.transform = 'scale(1.5)');
+
+// ON DOCUMENT LOAD AN ALERT WILL APEAR TELLING THE USER THE 9 OTHER THINGS THEY CAN DO WITH THE SITE
+window.addEventListener("load", function(event) {
+  alert("All resources finished loading! Ready for some Fun?!?!?! Try scrolling the page, try hovering over the nav logo heading, try dblclicking the nav links, try dragging the funbus banner image, try pressing a key on your keyboard...but wait...where did the navbar go?!?!?...thats ok, just hover over anywhere inside it to bring it back! Focus the name field in the form to see its background color change and leave its focus, aka blur event, to remove the background color. Input something into the email field then highlight it and see what happens. Try resizing the window and see what happens. Click on the Fun In The Sun Packages div and look in the console to see that you clicked it...do the same with the Sign Me Up button inside that div and notice it console logs its own message without console logging that the div was clicked...thats the power of the stopPropagation method...and thats all folks!!!");
+});
+
+const sunPackage = document.querySelector("#sun");
+const mountainPackage = document.querySelector("#mountain");
+const islandPackage = document.querySelector("#island");
+const radios = document.querySelector('fieldset');
+// radios.addEventListener('select', radios.style.backgroundColor = "red");
+
+radios.addEventListener("select", (e) => {e.target.style.backgroundColor = 'red'}, false);
+
+const nameInput = document.querySelector("#nameForm");
+const emailForm = document.querySelector("#emailForm");
+const phoneForm = document.querySelector("phoneForm");
+const specialInstructionsForm = document.querySelector("#specialForm");
+
+// ALERTS DONT DELETE ME WHEN SELECTING TEXT IN THE EMAIL FORM
+emailForm.addEventListener("select", ()=> alert("DONT DELETE ME!!!!"))
+// ADDS A BACKGROUND COLOR THE THE NAME INPUT WHEN IT IS FOCUSED
+nameInput.addEventListener("focus", (e) => {
+  nameInput.style.backgroundColor = "orangered"});
+// REMOVES THE BACKGROUND COLOR ON THE NAME INPUT WHEN FOCUS IS LEFT...I.E. ON BLUR EVENT
+nameInput.addEventListener('blur', (e) => {
+  nameInput.style.backgroundColor = "inherit"});
+
```

Comment:
Events look really good. Maybe try making some events using querySelectorAll and using your forEach to iterate over the nodelist that it gives you.